ffprobe_test.go

```diff
@@ -137,3 +137,35 @@ func validateData(t *testing.T, data *ProbeData) {
 		t.Errorf("this video starts at 0s.")
 	}
 }
+
+func Test_ProbeSideData(t *testing.T) {
+	ctx, cancelFn := context.WithTimeout(context.Background(), 3*time.Second)
+	defer cancelFn()
+
+	fileReader, err := os.Open("assets/test.mov")
+	if err != nil {
+		t.Errorf("Error opening test file: %v", err)
+	}
+
+	data, err := ProbeReader(ctx, fileReader)
+	if err != nil {
+		t.Errorf("Error getting data: %v", err)
+	}
+
+	videoStream := data.FirstVideoStream()
+	if videoStream == nil {
+		t.Error("Video Stream was nil")
+	}
+
+	if len(videoStream.SideDataList) < 1 {
+		t.Error("Side data list was empty")
+	}
+
+	rotation, err := videoStream.SideDataList[0].GetString("rotation")
```

Comment:
Rotation is an int field right? So maybe use `GetInt` ?

Reply:
Oops yep you're right